geps/gep-1713/index.md

```diff
@@ -115,6 +115,16 @@ type ListenerSetSpec struct {
 	// 2. ListenerSet ordered by creation time (oldest first)
 	// 3. ListenerSet ordered alphabetically by “{namespace}/{name}”.
 	//
+	// Regarding Conflict Management, Listeners in a ListenerSet follow the same
```

Comment:
Should we add this new clause to the API types godoc?

Also we might want to call out sectionName behaves a bit differently where sibling ListenerSets can have the same sectionName.

Reply:
so, per Nick's suggestion I have used a similar approach of https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/gateway_types.go#L72 that contains the full description on godoc.

Reply:
Does this contradict the GEP ?

https://gateway-api.sigs.k8s.io/reference/spec/#gatewayspec
The implementation MUST NOT pick one conflicting Listener as the winner.
ALL indistinct Listeners must not be accepted for processing.

https://gateway-api.sigs.k8s.io/geps/gep-1713/#listener-validation
Implementations MUST treat the parent Gateways as having the merged list of all listeners from itself and attached ListenerSets and validation of this list of listeners MUST behave the same as if the list were part of a single Gateway with the relaxed listener name constraints.

Reply:
```suggestion
	// Regarding Conflict Management, Listeners within in a ListenerSet follow the same
```

Reply:
I have at least removed it from the CRD generation, closing this for now